src/main/java/com/greedobank/reports/config/Configuration.java

```diff
@@ -0,0 +1,26 @@
+package com.greedobank.reports.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.web.client.RestTemplate;
+
+@org.springframework.context.annotation.Configuration
+public class Configuration {
```

Comment:
I believe self-made configurations should have specific names like `WebConfiguration`/`ApplicationConfiguration`